pkgs/applications/blockchains/zcash/librustzcash/default.nix

```diff
@@ -18,7 +18,7 @@ rustPlatform.buildRustPackage rec {
 
   installPhase = ''
     mkdir -p $out/lib
-    cp target/release/librustzcash.a $out/lib/
+    cp $releaseDir/librustzcash.a $out/lib/
```

Comment:
I see a handful more that need this:

```
$ git grep target/release
target/release
pkgs/applications/blockchains/zcash/librustzcash/default.nix:18:    cp target/release/librustzcash.a $out/lib/
pkgs/applications/misc/alacritty/default.nix:91:    install -D target/release/alacritty $out/bin/alacritty
pkgs/applications/misc/alacritty/default.nix:95:    cp -r target/release/osx/Alacritty.app $out/Applications/Alacritty.app
pkgs/applications/networking/dyndns/cfdyndns/default.nix:22:    cp -p target/release/cfdyndns $out/bin/
pkgs/applications/version-management/git-and-tools/git-ignore/default.nix:27:    cp target/release/build/git-ignore-*/out/git-ignore.1 "$man/man/man1/"
pkgs/development/tools/misc/ycmd/default.nix:73:    TARGET=$out/lib/ycmd/third_party/racerd/target/release
pkgs/development/tools/rust/racerd/default.nix:35:    cp -p target/release/racerd $out/bin/
pkgs/tools/misc/broot/default.nix:26:    OUT_DIR=target/release/build/broot-*/out
pkgs/tools/misc/fd/default.nix:21:    (cd target/release/build/fd-find-*/out
pkgs/tools/misc/lsd/default.nix:17:    install -Dm644 -t $out/share/zsh/site-functions/ target/release/build/lsd-*/out/_lsd
pkgs/tools/misc/lsd/default.nix:18:    install -Dm644 -t $out/share/fish/vendor_completions.d/ target/release/build/lsd-*/out/lsd.fish
pkgs/tools/misc/lsd/default.nix:19:    install -Dm644 -t $out/share/bash-completion/completions/ target/release/build/lsd-*/out/lsd.bash
pkgs/tools/misc/vdirsyncer/default.nix:58:    mkdir -p rust/target/release
pkgs/tools/misc/vdirsyncer/default.nix:59:    ln -s ${native}/lib/libvdirsyncer_rustext* rust/target/release/
pkgs/tools/networking/tox-node/default.nix:27:    install -D target/release/tox-node $out/bin/tox-node
pkgs/tools/text/ripgrep/default.nix:33:    (cd target/release/build/ripgrep-*/out
```

Might be worth flushing them out in a separate PR first, since it's a general improvement to the robustness of packaging unrelated to your specific change here.  Though doing it all in 1 bigger PR is probably fine too.  

I think these packages breaking due to your change is nice side-effect, since it flushes out non-robust package assumptions!

Reply:
I've opened https://github.com/NixOS/nixpkgs/pull/82982 to deal with (most of) these.

Reply:
Just merged the PR of @cole-h (thanks a lot btw!). Will wait until the changes are on staging and then do a rebase to get all of those packages fixed.